### DISCUSSION TITLE
How to List Top 100 Chunks Retrieved from Pinecone

### DISCUSSION THREAD

**Discussion by mikehudson2** [FlowiseAI/Flowise#1924] [Q&A]
I want to query a Pinecone index with a TopK of 100 and save all 100 chunks to a local file text/CSV/excel file or failing that simply display the chunks on the screen. Obviously I don't want the chunks passed to the LLM as that would congest/fill its context window.  How might I do this?

**Comment by HenryHengZJ** [comment]
You can't for now. But we do have some POC that enable you to do this, will update soon!
